tower-http/src/validate_request.rs

```diff
@@ -379,12 +379,7 @@ where
             .headers()
             .get_all(header::ACCEPT)
             .into_iter()
-            .flat_map(|header| {
-                header
-                    .to_str()
-                    .ok()
-                    .into_iter()
-            })
+            .flat_map(|header| header.to_str().ok().into_iter())
```

Comment:
This does the same thing and is easier to understand at a glance.

```suggestion
            .filter_map(|header| header.to_str().ok())
```